snapshot_tests/shader/base_params_output_resolution.wgsl

```diff
@@ -0,0 +1,40 @@
+struct VertexInput {
```

Comment:
I would prefer if those shaders were named base on what they are doing. If you want to mention in the name that it relies on output_resolution then it's fine to mention it.